### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by herbig**
## Description

Various bug fixes.  See commit history for descriptions.

## Issue / Notion doc (if applicable)

closes https://github.com/decent-dao/fractal-interface/issues/813
closes https://github.com/decent-dao/fractal-interface/issues/819
closes https://github.com/decent-dao/fractal-interface/issues/817
closes https://github.com/decent-dao/fractal-interface/issues/820
closes https://github.com/decent-dao/fractal-interface/issues/764

### CODE REVIEW

File: src/components/Proposals/MultisigProposalDetails/TxActions.tsx [hunk 1]
```diff
@@ -27,7 +27,10 @@ export function TxActions({ proposal }: { proposal: MultisigProposal }) {
 
   const [asyncRequest, asyncRequestPending] = useAsyncRequest();
   const [contractCall, contractCallPending] = useTransaction();
-  const isExecuted = proposal.state === TxProposalState.Executed;
+  const hasActions =
+    proposal.state === TxProposalState.Active ||
+    proposal.state === TxProposalState.Approved ||
+    proposal.state === TxProposalState.Queued;
```
Comment (herbig) [inline src/components/Proposals/MultisigProposalDetails/TxActions.tsx:33]: @mudrila @Da-Colon are there other states for a multi sign transaction that should have actions to perform here?

File: src/components/ui/Header/Avatar.tsx [hunk 2]
```diff
@@ -26,10 +26,7 @@ function URLAvatar({ url, size }: { size: AvatarSize; url: string }) {
   });
 
   return (
-    <Box
-      h={avatarSizes[size]}
-      w={avatarSizes[size]}
-    >
+    <Box w={avatarSizes[size]}>
```
Comment (herbig) [inline src/components/ui/Header/Avatar.tsx:29]: removing the height param on this box fixed not centering a rectangular ENS avatar

before
Screenshot 2022-12-18 at 12 32 27 AM

after

Screenshot 2022-12-18 at 12 32 18 AM

and no avatar is unchanged:

Screenshot 2022-12-18 at 12 31 45 AM
Screenshot 2022-12-18 at 12 31 39 AM

**Review by mobiusTripper-crypto — APPROVED** [review]

File: src/components/Proposals/MultisigProposalDetails/TxActions.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/Proposals/MultisigProposalDetails/TxActions.tsx:33]: at the moment no sir, in the future we might add a 'reject' action

**Review by Da-Colon — APPROVED** [review]
LGTM

**Pull request merged**